### PR TITLE
adding gen sprite, adjusting components to use old icon names

### PR DESCRIPTION
**Pull request by samageloff**

### CODE REVIEW

**Comment by samageloff** [comment]
[(2) React:ions Implement Sprite Renderer](https://trello.com/c/iA8se2WY/510-2-reactions-implement-sprite-renderer)

File: scripts/gen-sprite.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  materialDesignIconsPath + '/content/svg/production/ic_clear_24px.svg#icon-check-circle-2-1',
+  materialDesignIconsPath + '/content/svg/production/ic_content_paste_24px.svg#icon-clipboard-1',
+  materialDesignIconsPath + '/navigation/svg/production/ic_close_24px.svg#icon-delete-1',
+  materialDesignIconsPath + '/navigation/svg/production/ic_close_24px.svg#icon-delete-1-1',
+  materialDesignIconsPath + '/navigation/svg/production/ic_cancel_24px.svg#icon-delete-3',
+  materialDesignIconsPath + '/navigation/svg/production/ic_check_24px.svg#icon-check-1-1',
+	materialDesignIconsPath + '/action/svg/production/ic_info_outline_24px.svg#icon-information',
+	materialDesignIconsPath + '/communication/svg/production/ic_mail_outline_24px.svg#icon-mail-1',
+	materialDesignIconsPath + '/action/svg/production/ic_highlight_off_24px.svg#icon-highlight-off',
+	customIconsPath + '/icon-caret-bottom.svg#icon-caret'
```
Comment (jivinivan) [inline scripts/gen-sprite.js]: Fix indentation

File: scripts/gen-sprite.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+addFixtureFiles(spriter, iconList).compile({
+	symbol: {
+		sprite: 'sprite.svg'
+	}
+}, function (error, result) {
+	for (var type in result.symbol) {
+		mkdirp.sync(path.dirname(result.symbol[type].path));
+		fs.writeFileSync(result.symbol[type].path, result.symbol[type].contents);
+	}
+})
```
Comment (jivinivan) [inline scripts/gen-sprite.js]: Fix indentation

File: scripts/gen-sprite.js (hunk 1 above)
Comment (samageloff) [reply scripts/gen-sprite.js]: ⚡️

**Pull request merged**